### PR TITLE
user should be able to initialize rabbitmq on a new node using new UI

### PR DESCRIPTION
**Pull request by deepikasl**
https://github.com/Shippable/admiral/issues/1581

screenshot from 2018-01-12 17 43 09

screenshot from 2018-01-12 17 44 13

### CODE REVIEW

**Comment by ric03uec** [comment]
i dont get the design. in other places we're using a dropdown to select the node type, why are we using a radio button here?

**Comment by ric03uec** [comment]
also, is this a modal or is this being shown in the main panel?

**Comment by deepikasl** [comment]
this is the popup modal design, as per the [PM](https://github.com/Shippable/pm/issues/9817)

File: static/scripts/dashboard/dashboard2Ctrl.js [hunk 1]
```diff
@@ -1170,6 +1170,35 @@
...
+        var hasErr = '';
+
+        if ($scope.vm.initializeForm.msg.initType === 'new') {
+          hasErr = validateIP ($scope.vm.initializeForm.msg.address);
+          if (!$scope.vm.initializeForm.msg.confirmCommand) {
+            hasErr = 'has-error';
+          }
+        }
+        if ($scope.vm.initializeForm.msg.initType === 'new') {
+          hasErr = validateIP ($scope.vm.initializeForm.msg.password);
```
Comment (priyakirankumar) [inline static/scripts/dashboard/dashboard2Ctrl.js]: Use validatePassword instead of validateIP

**Pull request merged**